auth0/resource_auth0_log_stream.go

```diff
@@ -132,6 +132,18 @@ func newLogStream() *schema.Resource {
 					},
 				},
 			},
+			"aws_partner_event_source": {
```

Comment:
I think this omission is why you don't get the partner topic back?

Reply:
@mcalster yes, adding this into the `sink` object does return the values, but they're not easily accessible.  I've gone into more detail in your PR to the upstream repo.